community/asking-questions.md

```diff
@@ -0,0 +1,12 @@
+# Asking Questions on the Forum
```

Comment:
Maybe also a section about asking for complete mods? E.g. describe thoroughly, include mockups…

Reply:
Ok.